src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -884,22 +902,41 @@ private boolean updateSchema(Connection connection) throws Exception
 
             if (logger.isInfoEnabled())
             {
-                logger.info("Create scripts executed in "+(System.currentTimeMillis()-start)+" ms");
+                logger.info("Create scripts executed in " + (System.currentTimeMillis() - start) + " ms");
             }
         }
         else
         {
+            long start = System.currentTimeMillis();
+
             // Execute any pre-auto-update scripts
             checkSchemaPatchScripts(connection, preUpdateScriptPatches, true);
             
             // Execute any post-auto-update scripts
             checkSchemaPatchScripts(connection, postUpdateScriptPatches, true);
+
+            if (logger.isInfoEnabled())
+            {
+                logger.info(
+                    "Update case: checkSchemaPatchScripts scripts executed in " + (System.currentTimeMillis() - start) + " ms");
+            }
         }
 
+        ensureCurrentClusterMemberIsBootstrapping(connection);
+
         // Initialise Activiti DB, using an unclosable connection
         boolean activitiTablesExist = checkActivitiTablesExist(connection);
+        if (logger.isInfoEnabled())
+        {
+            logger.info("Activiti tables exist: " + activitiTablesExist);

```

Comment:
The Activiti tables should exist on each of installation. Maybe change this one to `debug`?